api/s2n.h

```diff
@@ -2162,9 +2164,13 @@ extern int s2n_connection_set_session(struct s2n_connection *conn, const uint8_t
 /**
  * Serializes the session state from connection and copies into the `session` buffer and returns the number of copied bytes
  *
- * The output of this function depends on whether session ids or session tickets are being used for resumption.
+ * The output of this function depends on whether session ids or session tickets are being used for resumption. If the first 
+ * byte in **session** is 1, then the next 2 bytes will contain the session ticket length, followed by session ticket and
+ * session state. In versions TLS1.3 and greater, (which allows multiple session tickets), the most recent session ticket
+ * received will be used. Note that the size of the session tickets varies.
+ * If the first byte in **session** is 0, then the next byte will contain session id length, followed by session id and session state.
  *
- * @note This is for < TLS 1.3 session resumption.
+ * @note This not recommended for > TLS 1.2 session resumption.
```

Comment:
can you elaborate what is not recommended. not calling this function?

Reply:
Also, you lost your verb. "is not recommended".